### PR TITLE
fixed-make-install-dependences

### PR DESCRIPTION
**Pull request by tudang**
I tried to build connectal from source on ubuntu 15.04 but it failed.
Replacing libgmp3c2 by libgmpxx4ldbl fixed the issue.

### CODE REVIEW

**Comment by hanw** [comment]
We need to be careful about backward compatibility, there needs to be a check on the version of ubuntu to install the correct package.

**Comment by hanw** [comment]
@jameyhicks

File: Makefile [hunk 1]
```diff
@@ -116,7 +116,7 @@ ifeq ($(shell uname), Darwin)
 	easy_install ply
 else
 	apt-get install asciidoc python-dev python-setuptools python-ply
-	apt-get install libgmp3c2
+	apt-get install libgmpxx4ldbl
```
Comment (jameyhicks) [inline Makefile]: libgmpxx4ldbl does not provide libgmp.so.3, which is required by the Bluespec compiler.

libgmp3c2 is no longer available in Ubuntu, so it's not much use having it there.

We currently install libgmp10 and then add a symbolic link:
    ln -s libgmp.so /usr/lib/x86_64-linux-gnu/libgmp.so.3

If you'd like to revise this pull request to do this instead, I would appreciate it.

I highly recommend installing the connectal package in order to install connectal and its dependences, so extra bonus points for creating a libgmp3c2-compatibility debian package, or maybe connectal-libgmp3c2, which creates the link.

**Comment by tudang** [comment]
I replace previous change by creating a link for libgmp.so.3 as your suggestion. The code check whether libgmp.so.3 exists. If not, create a link libgmp.so -> libgmp.so.3

**Comment by hanw** [comment]
This patch does not install libgmp10.

**Comment by hanw** [comment]
how about `ln -sf /usr/lib/x86_64-linux-gnu/libgmp.so /usr/lib/x86_64-linux-gnu/libgmp.so.3`

**Comment by tudang** [comment]
Thank you, Han. I've included libgmp10 package.

**Comment by jameyhicks** [comment]
Thanks for submitting this pull request!

**Pull request merged**